libs/utils/utils.js

```diff
@@ -567,3 +567,15 @@ export function createIntersectionObserver({ el, callback, once = true, options
   io.observe(el);
   return io;
 }
+
+export const getSectionMetadata = (el) => {
+  if (!el) return {};
+  const metadata = {};
+  el.childNodes.forEach((node) => {
+    const key = node.children?.[0]?.textContent?.toLowerCase();
+    if (!key) return;
+    const val = node.children?.[1]?.textContent?.toLowerCase();
```

Comment:
The metadata value may be case sensitive.  Probably should remove `toLowerCase()`

Reply:
It can? I would think we need to flatten that to all lowercase on purpose. We do that in section-metadata.

Reply:
This shouldn't even be here. This should be in the section metadata js.

Reply:
re: case sensitive, yeah you're right, I was also thinking of my card metadata use case which does need to have case preserved

Reply:
This code is to grab the section metadata values from the dom if the section-metadata.js isn't done so it doesn't really make sense to add it to that file. I originally had it in card.js but @chrischrischris said he had a similar use case and to add it to utils. If this doesn't work for Chris P, then it seems like maybe I should move it back to card.js

Reply:
section-metadata should be using this feature, and there's nothing wrong with asking section-metadata for it. We don't want block specific stuff in utils. There's nothing wrong with one block asking another block for info.

Reply:
Also, this is not a very commonly used feature, so our visitors paying the bandwidth for the code doesn't seem right.

Reply:
Updated